### PR TITLE
Add ignore for AMQP no route to host warning

### PR DESCRIPTION
**Pull request by krauselukas**

### CODE REVIEW

**Comment by okurz** [comment]
I doubt that is an error we want to ignore. And also we should at best ignore warns, not errors

**Comment by nicksinger** [comment]
@okurz honestly, it pollutes our log reports and does not show a real problem, does it? Even if the AMQP connection would die permanently and this message appears I would probably ignore it since it appears this often in our logs.
However - we need to improve the log output here (maybe with the same change, @mudler did for other messages -> compress them into one message per hour at max?) **or/and** fix the network issues we have there. Until then we've a #poo to track it - In my understanding the exact workflow we used to follow with this script.

Besides that, this is a warning not an error ;)

File: in.log [hunk 1]
```diff
@@ -52,3 +52,4 @@ more output that is not properly prefixed
 [Wed Aug  9 09:31:41 2017] [websockets:warn] A message received from unknown worker connection
 [Wed Aug  2 13:49:10 2017] [28861:warn] Stopping worker 4210 immediately
 [Wed Aug 23 09:49:19 2017] [4087:error] org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
+[Sun Aug 27 18:09:23 2017] [25795:warn] AMQP connection error: No route to host
```
Comment (okurz) [inline in.log]: why does it say "warn] …error"? Hm, I guess an error in a subcomponent can be considered a warning on top level, fine then

Comment (nicksinger) [reply in.log]: Pretty sure this is just a message from the AMQP lib we use which is then processed as warning

Comment (okurz) [reply in.log]: you guessed wrong: https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Plugin/AMQP.pm#L112

Comment (nicksinger) [reply in.log]: Interesting but just one more example on how we can improve log output from openqa :) thx for the research!

**Pull request merged**